orchestrator/domain/base.py

```diff
@@ -977,10 +970,7 @@ def depends_on(self) -> List[SubscriptionInstanceTable]:
 class ProductModel(BaseModel):
     """Represent the product as defined in the database as a dataclass."""
 
-    class Config:
-        validate_assignment = True  # pragma: no mutate
-        validate_all = True  # pragma: no mutate
-        arbitrary_types_allowed = True  # pragma: no mutate
```

Comment:
no need to add this to the ConfigDict?

Reply:
yea, its for allowing classes that are not extended from pydantic `BaseModel`, don't think its necessary.
https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.arbitrary_types_allowed